projects/fhi-angular-highcharts/src/styles/blocks/_fhi-highcharts-chart.block.scss

```diff
@@ -1,14 +1,56 @@
+$golden-ratio: 1.618;
+$golden-ratio-conjugate: 0.618;
+$square: 1;
+
 .fhi-highcharts-chart {
-    @include diagram-container;
-    @include chart-in-fullscreen-modal;
+    width: 100%;
+    container-type: inline-size;
 
     highcharts-chart {
-        @include fill-container;
+        display: block;
+        position: relative;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        max-height: 90vh;
+        aspect-ratio: $square;
+    }
+    &--map {
+        highcharts-chart {
+            aspect-ratio: $golden-ratio-conjugate;
+        }
+    }
+}
+
+@container (width > 350px) {
+    .fhi-highcharts-chart--map {
+        highcharts-chart {
+            aspect-ratio: $square;
+        }
+    }
+}
+
+@container (width > 650px) {
+    .fhi-highcharts-chart {
+        highcharts-chart {
+            aspect-ratio: $golden-ratio;
+        }
+    }
+}
+
+@container (width > 650px) {
```

Comment:
Can this `@container (width > 650px)` block be merged with the one above?

Reply:
Absolutely! They started out with different widths 😉